models/topics.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.fetchAllTopics = () => {
+  return db
+  .query(
+    `
+      SELECT * FROM topics;
+  `
+  )
+  .then((topics) => {
```

Comment:
I might avoid the name `topics` here as it is not an array of your topic objects, rather the results object